### DISCUSSION TITLE
How to order mpdf index entries by page number not an alphabetical order?

### DISCUSSION THREAD

**Discussion by vijayasanthi1993** [mpdf/mpdf#1723] [Q&A] [answered]
I just want to order the mpdf index page by page number.
The current mpdf output is,
Index
A 2
ACCOUNTING POLICIES 10
DETAILED INCOME STATEMENT 14
INDEPENDENT REVIEWER’S REPORT 3
NOTES TO THE FINANCIAL STATEMENTS 13
STATEMENT OF CASH FLOWS 9
STATEMENT OF CHANGES IN EQUITY 8
STATEMENT OF COMPREHENSIVE INCOME 7
STATEMENT OF FINANCIAL POSITION 6

But i want this output
Index
A 2
INDEPENDENT REVIEWER’S REPORT 3
STATEMENT OF FINANCIAL POSITION 6
STATEMENT OF COMPREHENSIVE INCOME 7
STATEMENT OF CHANGES IN EQUITY 8
STATEMENT OF CASH FLOWS 9
ACCOUNTING POLICIES 10
NOTES TO THE FINANCIAL STATEMENTS 13
DETAILED INCOME STATEMENT 14

Is anybody know the answer please guide us

**Comment by finwe** [comment, marked as answer]
That's not an index, that's table of contents https://mpdf.github.io/what-else-can-i-do/table-of-contents.html

**Reply by vijayasanthi1993** [reply]
Yes it's working. Thanks for your reply.

**Reply by finwe** [reply]
No, TOC is always generated at the end so that it itself doesn't mess with page numbers.

**Reply by vijayasanthi1993** [reply]
Okay. Thank you

**Comment by vijayasanthi1993** [comment]
TOCpagebreak - Added extra blank pages #1730

I tried to add a TOC but adds the blank pages
